storage_service/README.md

```diff
@@ -75,6 +76,11 @@ curl -X POST http://localhost:8080/manifests/my-manifest-456 \
 curl -X POST http://localhost:8080/manifests/my-manifest-789?manifest_type=video \
   -H "Content-Type: application/octet-stream" \
   --data-binary @manifest.bin
+
+# C2PA URN format (automatically validated)
```

Comment:
Automatically validated where / in which part of the data flow?

Reply:
It’s in main.rs 

```rust
fn is_valid_manifest_id(id: &str) -> bool {
    atlas_common::validation::validate_manifest_id(id).is_ok()
}
```
But let me remove from README since we are validating all of id’s, so it’s nothing special.